app/static/js/trackVolunteers.js

```diff
@@ -21,10 +30,26 @@ $(document).ready(function() {
       });
     });
 
+    function updateSelectVolunteer(){
+      let checkboxlist = $("#addVolunteerModal input[type=checkbox]")
+      var shouldEnableButton = false
+      $.each(checkboxlist, function(index, checkbox){
+          if(checkbox["checked"] == true){
+            shouldEnableButton = true
+          }
+      })
+      if (shouldEnableButton){
+        $("#addVolunteersButton").prop("disabled", false)
+      }
+      else{
+        $("#addVolunteersButton").prop("disabled", true)
+      }
```

Comment:
I would combine all of this into the loop. Instead of using `shouldEnableButton` you could just do the actual property setting in place of it.

Reply:
If you wanted to get really fancy, this whole bit could be two pieces, a reduce to get a single boolean value, and a property setting based on that boolean. This isn't necessarily a recommendation, but might be fun.